Modules/Core/SpatialObjects/include/itkSpatialObjectProperty.h

```diff
@@ -91,18 +91,12 @@ class ITKSpatialObjects_EXPORT SpatialObjectProperty
   GetAlpha() const;
 
   void
-  SetName(const std::string & name)
+  SetName(std::string name)
   {
     m_Name = name;
```

Comment:
If you _do_ pass std::string by-value, please consider `std::move` here, to avoid extra memory reallocation:

```cpp
    m_Name = std::move(name);
```

Reply:
Cool idea.  Will do in as an amend :)